src/hoprd_jobs.rs

```diff
@@ -178,45 +178,47 @@ fn build_create_node_args (secret_name: &String) -> Vec<String> {
 /// Create the args command line to register a node
 fn build_register_node_args (_secret_name: &String) -> Vec<String> {
     let mut args = Vec::with_capacity(1);
-    args.push("/app/hoprnet/.cargo/bin/foundry-tool".to_owned());
-    args.push("network-registry".to_owned());
-    args.push("--environment-name".to_owned());
-    args.push("${HOPRD_ENVIRONMENT}".to_owned());
-    args.push("--environment-type".to_owned());
-    args.push("${HOPRD_ENVIRONMENT_TYPE}".to_owned());
-    args.push("--peer-ids".to_owned());
-    args.push("${HOPRD_PEER_ID}".to_owned());
-    args.push("--private-key".to_owned());
-    args.push("${PRIVATE_KEY}".to_owned());
-    args.push("--make-root".to_owned());
-    args.push("\"../contracts\"".to_owned());
+    args.push("sleep 2".to_owned());
+    // args.push("/app/hoprnet/.cargo/bin/hopli".to_owned());
+    // args.push("network-registry".to_owned());
+    // args.push("--environment-name".to_owned());
+    // args.push("${HOPRD_ENVIRONMENT}".to_owned());
+    // args.push("--environment-type".to_owned());
+    // args.push("${HOPRD_ENVIRONMENT_TYPE}".to_owned());
+    // args.push("--peer-ids".to_owned());
+    // args.push("${HOPRD_PEER_ID}".to_owned());
+    // args.push("--private-key".to_owned());
+    // args.push("${PRIVATE_KEY}".to_owned());
+    // args.push("--make-root".to_owned());
+    // args.push("\"../contracts\"".to_owned());
```

Comment:
Intended to left commented out?

Reply:
Waiting for Hopli PR to be merged, right.